sequencing-server/src/lib/mustache/util/time.ts

```diff
@@ -96,100 +80,103 @@ function checkNumDurationComponents(split: string[]): split is [string, string,
 
 export function AERIEDurationToISO8601(duration: string): string {
   // HHHHHH...:MM:SS.mmmuuu -> PHHMMSS.mmmuuuS
-  duration = duration.replace("Z", "");
-  let split = duration.split(":")
-  let isNegative = duration.includes("-")
+  duration = duration.replace('Z', '');
+  let split = duration.split(':');
+  let isNegative = duration.includes('-');
 
-  if (!checkNumDurationComponents(split)) { // required so editor wouldn't flag a type error
+  if (!checkNumDurationComponents(split)) {
+    // required so editor wouldn't flag a type error
     throw new Error(`Invalid duration string: ${duration}`);
   }
 
-  let hours = parseInt(split[0].replace("-", ""))
-  let minutes = parseInt(split[1])
-  let split2 = (split[2]).split(".")
-  let seconds = parseInt(split2[0] as string)
+  let hours = parseInt(split[0].replace('-', ''));
+  let minutes = parseInt(split[1]);
+  let split2 = split[2].split('.');
+  let seconds = parseInt(split2[0] as string);
   if (split2.length > 1) {
-    let microseconds = parseInt(split2[1]?.padEnd(6, "0") as string)
-    let microsecondsString = `${microseconds}`.padStart(6, "0")
-    return `${isNegative ? "-" : ""}PT${hours > 0 ? `${hours}H` : ""}${minutes > 0 ? `${minutes}M` : ""}${seconds > 0 && microseconds > 0 ? `${seconds}.${microsecondsString}S` : (seconds > 0) ? `${seconds}$` : (microseconds > 0) ? `0.${microsecondsString}S` : ""}`
-  }
-  else {
-    return `${isNegative ? "-" : ""}PT${hours > 0 ? `${hours}H` : ""}${minutes > 0 ? `${minutes}M` : ""}${seconds > 0 ? `${seconds}S` : ""}`
+    let microseconds = parseInt(split2[1]?.padEnd(6, '0') as string);
+    let microsecondsString = `${microseconds}`.padStart(6, '0');
+    return `${isNegative ? '-' : ''}PT${hours > 0 ? `${hours}H` : ''}${minutes > 0 ? `${minutes}M` : ''}${
+      seconds > 0 && microseconds > 0
```

Comment:
My only commentary here is that this code formatting looks a little bizarre, but it seems completely solid!